.github/workflows/R-CMD-check.yaml

```diff
@@ -0,0 +1,87 @@
+# For help debugging build failures open an issue on the RStudio community with the 'github-actions' tag.
```

Comment:
we have to switch to GitHub Actions cf https://ropensci.org/technotes/2020/11/19/moving-away-travis/

Reply:
FYI we are gradually switching to workflows that use pak instead of remotes. But this is still experimental, i.e. we are experimenting on ourselves. Something to contemplate in the future. Example:

https://github.com/r-lib/usethis/blob/master/.github/workflows/R-CMD-check.yaml

Reply:
Good to know :eyes: